erpnext/stock/doctype/pick_list/pick_list.py

```diff
@@ -86,20 +97,16 @@ def update_so(self, so_item, picked_qty, item_code):
 				frappe.throw(
 					_(
 						"You are picking more than required quantity for {}. Check if there is any other pick list created for {}"
-					).format(item_code, so_doc.name)
+					).format(item_code, item.sales_order)
```

Comment:
Bug: in multi-uom cases, `picked_qty` is in stock UOM and `actual_qty` is in UOM specified in SO.
It always throws the error even though I am picking half qty
Eg: Item Double Dozen Eggs > Stock UOM: Nos > 1 Box = 24 Nos

Place SO for 10 Boxes (240 Nos) > Pick 5 Boxes (120 Nos) > It says I am overpicking

Reply:
Nice catch. Need to fetch conversion factor specified in SO row too ig 🤔

Reply:
Changed field to stock_qty for comparison and added tests: https://github.com/frappe/erpnext/pull/30762/commits/a740d4b7958c1174053c52c2621884bb1a666238 

I am bit conflicted about UOM of `picked_qty` field. So far it's been in stock_qty.

It's next to stock UOM field so maybe it's fine

<img width="988" alt="Screenshot 2022-04-27 at 12 39 05 PM" src="https://user-images.githubusercontent.com/9079960/165461421-91ed5f32-8ef8-41b5-951b-0d3a42d0d205.png">

Reply:
will update label of picked qty field to indicate it's in stock UOM.

Reply:
We usually back update in stock qty only, saves headache of converting. All % is also calculated w.r.t to stock qty only.